src/plugins/experimental-discord/update_discord_api_snapshots.sh

```diff
@@ -40,7 +40,7 @@ fetch "/guilds/$GUILD_ID"
 fetch "/guilds/$GUILD_ID/channels"
 fetch "/guilds/$GUILD_ID/roles"
 fetch "/guilds/$GUILD_ID/emojis"
-fetch "/guilds/$GUILD_ID/members?limit=1000"
+fetch "/guilds/$GUILD_ID/members?after=0&limit=1000"
```

Comment:
Hmm, if you needed to make a change to this file, you should re_run the snapshot fetcher too. since now one of the snapshot files will have a different name. surprised if this didn't cause any issues as written.

Reply:
I'm pretty sure I did run these, and I just tried running again and no changes were made. Strange.